presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriver.java

```diff
@@ -46,7 +63,13 @@
 
     private static final String USER_PROPERTY = "user";
 
-    private final QueryExecutor queryExecutor;
+    private final AtomicBoolean closed = new AtomicBoolean(false);
+
+    private final JettyIoPool jettyIoPool;
+    private final ScheduledExecutorService queryExecutorCleanupService;
+
+    private final LoadingCache<Map<Object, Object>, QueryExecutorHolder> queryExecutorCache;
```

Comment:
I would rather replace the map with the object, say `QueryExecutorCacheKey` or `QueryExecutorProperies`. Or, at least, please specify the `Key` and `Value` types explicitly instead of the Object. If you specify `Object` everyone can put everything it this map. Even the instances of the classes that doesn't have `Equal` and `HashCode` methods implements.